subiquity/server/controllers/filesystem.py

```diff
@@ -988,3 +991,8 @@ def make_autoinstall(self):
         if 'swap' in rendered:
             r['swap'] = rendered['swap']
         return r
+
+    async def autoinstall_applied_GET(self, wait: bool = False) -> None:
+        if wait:
+            await self.autoinstall_applied.wait()
+        return self.autoinstall_applied.is_set()
```

Comment:
er this returns bool, contrary to the type annotation.

Reply:
Done, thanks.